velox/functions/prestosql/URLFunctions.h

```diff
@@ -84,6 +85,35 @@ FOLLY_ALWAYS_INLINE void urlEscape(TOutString& output, const TInString& input) {
   output.resize(outIndex);
 }
 
+template <typename TInString>
+FOLLY_ALWAYS_INLINE bool urlUnescapeCheck(const TInString& input) {
```

Comment:
Would you add a comment to explain what checks are performed here?

naming: start with a verb (checkUrlUnescape) or isValidUrl